sys/linux/usbip.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+resource usbip_server_fd[fd]
+
+syz_usbip_server_init() usbip_server_fd
+
+syz_usbip_server_send(fd usbip_server_fd, buffer ptr[in, usbip_header], len bytesize[buffer])
+
+usbip_header_basic {
+	command		const[USBIP_RET_SUBMIT, int32be]
+	seqnum		int32be
+	devid		int32be
+	direction	flags[direction_flags, int32be]
+	ep		int32be
+}
+
+usbip_header_ret_submit {
+	status			int32be
+	actual_length		int32be
+	start_frame		int32be
+	number_of_packets	int32be[0:USBIP_MAX_ISO_PACKETS]
+	error_count		int32be
+}
+
+usbip_header {
+	base		usbip_header_basic
+	ret_submit	usbip_header_ret_submit
+}
+
+define USBIP_RET_SUBMIT	0x0003
```

Comment:
Does `include <drivers/usb/usbip/usbip_common.h>` not work? If it works, then do that instead of defining manually.